app/bundles/CampaignBundle/Command/WriteCountTrait.php

```diff
@@ -12,17 +12,15 @@ private function writeCounts(OutputInterface $output, TranslatorInterface $trans
     {
         $output->writeln('');
         $output->writeln(
-            '<comment>'.$translator->transChoice(
+            '<comment>'.$translator->trans(
                 'mautic.campaign.trigger.events_executed',
-                $counter->getTotalExecuted(),
                 ['%events%' => $counter->getTotalExecuted()]
```

Comment:
```suggestion
                ['%count%' => $counter->getTotalExecuted()]
```
This should have the `%count%` param. We can replace the `%events%` param with it as it holds the same data and then update it in the .ini file. What do you think?

Same for the one bellow.

Reply:
Yes, we can update all the required ini files with `%count%` parameter for consistency. This is the expectation for your comment, right?

Reply:
Yes. And the pluralization won't work for `%events%` param as it needs the `%count%` param